rest/routing.go

```diff
@@ -357,6 +363,14 @@ func CreateMetricRouter(sc *ServerContext) *mux.Router {
 	return r
 }
 
+func CreateDiagnosticRouter(sc *ServerContext) *mux.Router {
+	r := mux.NewRouter()
+	r.StrictSlash(true)
+	r.Handle("/_ping", makeSilentHandler(sc, publicPrivs, nil, nil, (*handler).handlePing)).Methods("GET", "HEAD")
```

Comment:
Let's create a common function for `_ping` that works for all interfaces on public. port and just calls this function, and we'll call it for each endpoint.